test/foundry/Regular_Tests/Fuzzing.t.sol

```diff
@@ -0,0 +1,247 @@
+        jetStaking.stake(x);
+        jetStaking.unstakeAll();
+        vm.warp(block.timestamp + 2 days + 1);
+        jetStaking.withdrawAll();
+        assert(balanceBefore == aurora.balanceOf(address(this)));
+    }
+
+    /**
+     * @dev Stake and unstake after waiting 1 week from the jetStaking contract. Due
+     *      to the one week waiting time rewards should have accrued.
+     *
+     * @param x Amount of aurora tokens to stake
+     */
+    function testStakeWaitAndWithdraw(uint256 x) public {
+        vm.assume(x > 0 && x < maxSupply);
+        uint256 balanceBefore = aurora.balanceOf(address(this));
+        aurora.approve(address(jetStaking), x);
+        jetStaking.stake(x);
+        vm.warp(block.timestamp + 1 weeks);
+        jetStaking.unstakeAll();
+        vm.warp(block.timestamp + 2 days + 1);
+        jetStaking.withdrawAll();
+        assert(balanceBefore < aurora.balanceOf(address(this)));
+    }
+
+    /**
+     * @dev Stake and unstake after waiting 1 week from the jetStaking contract. Batch
+     *      withdraw allows withdraw several stream tokens in one transaction. Here we
+     *      are testing the withdrawal of two streams (0 & 1), and making sure our balance
+     *      of both tokens has increased since the time of staking.
+     *
+     * @param x Amount of aurora tokens to stake
+     */
+    function testStakeAndBatchWithdraw(uint256 x) public {
+        vm.assume(x > 0 && x < maxSupply);
+        uint256 balanceBefore = aurora.balanceOf(address(this));
+        uint256 rewardBalance = rewardToken1.balanceOf(address(this));
+        aurora.approve(address(jetStaking), x);
+        jetStaking.stake(x);
+        vm.warp(block.timestamp + 1 weeks);
+        jetStaking.moveAllRewardsToPending();
+        jetStaking.unstakeAll();
+        vm.warp(block.timestamp + 2 days + 1);
+        uint256[] memory streamId = new uint256[](2);
+        streamId[0] = 0;
+        streamId[1] = 1;
+        jetStaking.batchWithdraw(streamId);
+        assert(balanceBefore < aurora.balanceOf(address(this)));
+    }
+
+    /*/////////////////////////////////////////////////////////////
+                    STAKE / UNSTAKE (PARAM: TIME)
+    /////////////////////////////////////////////////////////////*/
+
+    /**
+     * @dev Stake and unstake after waiting x time from the jetStaking contract. Due
+     *      to the waiting time rewards should have accrued.
+     *
+     * @param x Number of seconds to wait before unstaking
+     */
+    function testStakeWaitAndWithdraw2(uint256 x) public {
+        uint256 amount = 100 ether;
+        vm.assume(x > 0 && x < 315360000); // seconds in 10 years
+        uint256 balanceBefore = aurora.balanceOf(address(this));
+        aurora.approve(address(jetStaking), amount);
+        jetStaking.stake(amount);
+        vm.warp(block.timestamp + x);
+        jetStaking.unstakeAll();
+        vm.warp(block.timestamp + 2 days + 1);
+        jetStaking.withdrawAll();
+        if(x > 50) {
```

Comment:
Can you add a commend why `50` seconds ? if there is a reason for this number, we can move this constraint to the `vm.assume`.

Reply:
Just an arbitrary number